docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2.4'
+services:
+  postgres:
+    container_name: postgres
+    restart: always
+    image: eg_postgresql
+    volumes:
```

Comment:
We don't need persistence during the tests, so we can avoid exposing the volumes to the host

Reply:
Ok, but are we going to backup pg data on our tests? If so I think it would be interesting to expose a volume.

Reply:
You got a point, it makes sense (especially for a manual inspection of the backup content)